keyboards/rot13labs/h4ckb0ard/info.json

```diff
@@ -0,0 +1,153 @@
+            "solid_reactive_simple": true,
+            "solid_reactive": true,
+            "splash": true,
+            "multisplash": true,
+            "solid_splash": true,
+            "solid_reactive_cross": true
+        },
+        "driver": "ws2812",
+        "layout": [
+            {"flags": 1, "matrix": [0, 11], "x": 0, "y": 0},
+            {"flags": 1, "matrix": [0, 10], "x": 1, "y": 0},
+            {"flags": 1, "matrix": [0, 9], "x": 2, "y": 0},
+            {"flags": 1, "matrix": [0, 8], "x": 3, "y": 0},
+            {"flags": 1, "matrix": [0, 7], "x": 4, "y": 0},
+            {"flags": 1, "matrix": [0, 6], "x": 5, "y": 0},
+            {"flags": 1, "matrix": [0, 5], "x": 6, "y": 0},
+            {"flags": 1, "matrix": [0, 4], "x": 7, "y": 0},
+            {"flags": 1, "matrix": [0, 3], "x": 8, "y": 0},
+            {"flags": 1, "matrix": [0, 2], "x": 9, "y": 0},
+            {"flags": 1, "matrix": [0, 1], "x": 10, "y": 0},
+            {"flags": 1, "matrix": [0, 0], "x": 11, "y": 0},
+
+            {"flags": 1, "matrix": [1, 0], "x": 0, "y": 1},
+            {"flags": 1, "matrix": [1, 1], "x": 1.25, "y": 1},
+            {"flags": 1, "matrix": [1, 2], "x": 2.25, "y": 1},
+            {"flags": 1, "matrix": [1, 3], "x": 3.25, "y": 1},
+            {"flags": 1, "matrix": [1, 4], "x": 4.25, "y": 1},
+            {"flags": 1, "matrix": [1, 5], "x": 5.25, "y": 1},
+            {"flags": 1, "matrix": [1, 6], "x": 6.25, "y": 1},
+            {"flags": 1, "matrix": [1, 7], "x": 7.25, "y": 1},
+            {"flags": 1, "matrix": [1, 8], "x": 8.25, "y": 1},
+            {"flags": 1, "matrix": [1, 9], "x": 9.25, "y": 1},
+            {"flags": 1, "matrix": [1, 11], "x": 10.25, "y": 1},
+
+            {"flags": 1, "matrix": [2, 11], "x": 0, "y": 2},
+            {"flags": 1, "matrix": [2, 10], "x": 1.75, "y": 2},
+            {"flags": 1, "matrix": [2, 8], "x": 2.75, "y": 2},
+            {"flags": 1, "matrix": [2, 7], "x": 3.75, "y": 2},
+            {"flags": 1, "matrix": [2, 6], "x": 4.75, "y": 2},
+            {"flags": 1, "matrix": [2, 5], "x": 5.75, "y": 2},
+            {"flags": 1, "matrix": [2, 4], "x": 6.75, "y": 2},
+            {"flags": 1, "matrix": [2, 3], "x": 7.75, "y": 2},
+            {"flags": 1, "matrix": [2, 2], "x": 8.75, "y": 2},
+            {"flags": 1, "matrix": [2, 1], "x": 10.5, "y": 2},
+            {"flags": 1, "matrix": [2, 0], "x": 11.5, "y": 2},
+
+            {"flags": 1, "matrix": [3, 0], "x": 0, "y": 3},
+            {"flags": 1, "matrix": [3, 1], "x": 1.25, "y": 3},
+            {"flags": 1, "matrix": [3, 2], "x": 2.5, "y": 3},
+            {"flags": 1, "matrix": [3, 4], "x": 3.5, "y": 3},
+            {"flags": 1, "matrix": [3, 6], "x": 5.75, "y": 3},
+            {"flags": 1, "matrix": [3, 8], "x": 8.5, "y": 3},
+            {"flags": 1, "matrix": [3, 9], "x": 9.5, "y": 3},
+            {"flags": 1, "matrix": [3, 10], "x": 10.5, "y": 3},
+            {"flags": 1, "matrix": [3, 11], "x": 11.5, "y": 3}
```

Comment:
Every key being marked as a "modifier" here is incorrect.

Reply:
Sorry, im totally not familiar with what you mean by "modifier"? This is my first build with backlighting so i'm definitely still figuring it out. Is that the "flags": 1 part or just being in this list at all?

Reply:
`"flags"` key here maps to https://docs.qmk.fm/#/feature_rgb_matrix?id=flags

Reply:
Awesome, Thank you! I updated all of those to be 4 for backlighting.